docs/vdc/networking-virtual-datacenter.md

```diff
@@ -0,0 +1,397 @@
+
+Synchronization or heartbeat monitoring of applications in different vDCs requires communication between them. Two vDCs in different regions can be connected through:
+
+-   VNet Peering - VNet Peering can connect hubs across regions
+-   ExpressRoute private peering when the vDC hubs are connected to the same ExpressRoute circuit
+-   multiple ExpressRoute circuits connected via your corporate backbone and your vDC mesh connected to the ExpressRoute circuits
+-   Site-to-Site VPN connections between your vDC hubs in each Azure Region
+
+Usually VNet Peering or ExpressRoute connections are the preferred mechanism due higher bandwidth and consistent latency when transiting through the Microsoft backbone.
+
+There is no magic recipe to validate an application distributed between two (or more) different vDCs located in different regions. Customers should run network qualification tests to verify the latency and bandwidth of the connections and target whether synchronous or asynchronous data replication is appropriate and what the optimal recovery time objective (RTO) can be for your workloads.
+
+#### Mechanism to divert traffic between DC
+One effective technique to divert the traffic incoming in one DC to another is based on DNS. [Azure Traffic Manager][TM] uses the Domain Name System (DNS) mechanism to direct the end-user traffic to the most appropriate public endpoint in a specific vDC. Through probes, Traffic Manager periodically checks the service health of public endpoints in different vDCs and, in case of failure of those endpoints, it routes automatically to the secondary vDC.
+
+Traffic Manager works on Azure public endpoints and can be used, for example, to control/divert traffic to Azure VMs and Web Apps in the appropriate vDC. Traffic Manager is resilient even in the face of an entire Azure region failing and can control the distribution of user traffic for service endpoints in different vDCs based on several criteria (for instance, failure of a service in a specific vDC, or selecting the vDC with the lowest network latency for the client).
+
+### Conclusion
+The Virtual Data Center is an approach to data center migration into the cloud that uses a combination of features and capabilities to create a scalable architecture in Azure that maximizes cloud resource use, reducing costs, and simplifying system governance. The vDC concept is based on a hub-spokes topology, providing common shared services in the hub and allowing specific applications/workloads in the spokes. A vDC matches the structure of company roles, where different departments (Central IT, DevOps, operation and maintenance) work together, each with a specific list of roles and duties. A vDC satisfies the requirements for a "Lift and Shift" migration, but also provides many advantages to native cloud deployments.
+
+## References
```

Comment:
This table is a bit nonstandard - I'd delete this section, pick the top N (5?) to go in the Next Steps section.

If you keep the table, use standard markdown for the table headings.